ocaml/xapi/db_gc_util.ml

```diff
@@ -222,6 +222,23 @@ let gc_host_metrics ~__context =
         Db.Host_metrics.destroy ~__context ~self:hmetric)
     all_host_metrics
 
+let gc_certificates ~__context =
+  let all_certificates = Db.Certificate.get_all_records ~__context in
+  let get_certificates =
+    List.map (fun self -> Db.Host.get_certificates ~__context ~self)
+  in
+  let host_certificates =
+    get_certificates (Db.Host.get_all ~__context) |> List.flatten
+  in
```

Comment:
you can use List.concat_map, I'd also just do it in one line

Reply:
understand the eagerness, this needs to be backported to stockholm :p

Reply:
I'll change it in the backport ;p